src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String lineData;
+        int supplyAmount = 0;
+        int buyAmount = 0;
+
+        try (BufferedReader inputData = new BufferedReader(new FileReader(fromFileName))) {
+            while ((lineData = inputData.readLine()) != null) {
+                String[] values = lineData.split(",");
+                String operationType = values[0];
```

Comment:
Avoid magic numbers: Indexes should always have constants 

see https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file

Reply:
constants was added